client/lib/sites-list/list.js

```diff
@@ -38,9 +38,6 @@ function SitesList() {
 	this.selected = null;
 	this.lastSelected = null;
 	this.propagateChange = this.propagateChange.bind( this );
-
-	PreferencesActions.fetch();
-	this.recentlySelected = PreferencesStore.get( 'recentSites' ) || [];
```

Comment:
> I see the following errors when loading the stats page with empty localStorage

@rralian those are present in master and not related to this, it's just how the stats controller is validating the data.

Fixed the other issue during navigation, thanks.